Cargo.toml

```diff
@@ -1,15 +1,33 @@
+[workspace]
+members = ["extensions/*"]
```

Comment:
I generally prefer avoiding the glob workspace members because it does not play well with `git`. Since in some cases `git` does not delete the directories when checking out other branches, so cargo fails because it thinks the empty directory is a workspace member and expects a crate there.